package.json

```diff
@@ -22,8 +22,8 @@
     "method-override": "^2.3.7",
     "moment": "^2.10.3",
     "newrelic": "^1.40.0",
-    "passport": "^0.2.1",
-    "passport-github": "^0.1.5",
+    "passport": "0.3.2",
```

Comment:
upgraded `passport`

I tried to upgrade `passport-github` as well, but its internals changed in ways that break our tests (I think due to our usage of `nock`s), and I didn't want to deal with modifying all of them. Instead, I locked the version down.